<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caja</class>
 <widget class="QWidget" name="Caja">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>125</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>125</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabEditMode">
       <attribute name="title">
        <string>Edit Mode</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>823</width>
             <height>70</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxSize">
              <property name="currentText">
               <string>Tiny</string>
              </property>
              <item>
               <property name="text">
                <string>Tiny</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Small</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Normal</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Large</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Huge</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="Line" name="separator">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonBold">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/bold.png</normaloff>:/images/toolbar/bold.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+B</string>
              </property>
              <property name="arrowType">
               <enum>Qt::NoArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonItalic">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/italic.png</normaloff>:/images/toolbar/italic.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+I</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonUnderline">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/underline.png</normaloff>:/images/toolbar/underline.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+U</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonStrikethrough">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/strikethrough.png</normaloff>:/images/toolbar/strikethrough.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+T</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonSmallCaps">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/smallcaps.png</normaloff>:/images/toolbar/smallcaps.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+C</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonHyperlink">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/menu/link.png</normaloff>:/images/menu/link.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+K</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_Footnote">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/footnote.png</normaloff>:/images/toolbar/footnote.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="separator_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonJustify">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/justify.png</normaloff>:/images/toolbar/justify.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+Down</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonCenterAlign">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/centeralign.png</normaloff>:/images/toolbar/centeralign.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+Up</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonRightAlign">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/rightalign.png</normaloff>:/images/toolbar/rightalign.png</iconset>
              </property>
              <property name="shortcut">
               <string>Ctrl+Right</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="separator_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonItemize">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/itemize.png</normaloff>:/images/toolbar/itemize.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonEnumerate">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/enumerate.png</normaloff>:/images/toolbar/enumerate.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="separator_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonSymbol">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/menu/symbol.png</normaloff>:/images/menu/symbol.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonCite">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/menu/comment.png</normaloff>:/images/menu/comment.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonReference">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/images/toolbar/reference.png</normaloff>:/images/toolbar/reference.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabReviewMode">
       <attribute name="title">
        <string>Review Mode</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="4">
         <widget class="QLabel" name="labelStrikethroughRatioValue">
          <property name="text">
           <string>0 %</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QLabel" name="labelWarning">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="labelSmallCapsRatioValue">
          <property name="text">
           <string>0 %</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="labelItalicsRatioValue">
          <property name="text">
           <string>0 %</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="labelUnderlineRatioValue">
          <property name="text">
           <string>0 %</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelBoldRatioValue">
          <property name="text">
           <string>0 %</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLabel" name="labelStrikethroughRatio">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Strikethrough&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="labelSmallCapsRatio">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Small Caps&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="labelUnderlineRatio">
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Underline&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelItalicsRatio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Italics&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelBoldRatio">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Bold&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
